frameworks/g-cloud-9/questions/services/multiqAPIHosting.yml

```diff
@@ -1,18 +1,17 @@
+id: multiqAPI
```

Comment:
No, for multiquestions the names are currently used for slug generation. Would be good to reconcile this difference in the future, but we need to look at slugs as a larger piece of work, I think.